src/context/walletContext/walletContext.tsx

```diff
@@ -27,21 +35,20 @@ const WalletContext = createContext<WalletContextType | undefined>(undefined)
  * adding accounts and logging out. It also provides hooks for child components to access user data and auth-related functions.
  */
 export const WalletProvider = ({ children }: { children: ReactNode }) => {
+    const [promptWalletSigninOpen, setPromptWalletSigninOpen] = useState(false)
+    const { push } = useRouter()
     ////// ZeroDev props
-    const { address: kernelClientAddress, isKernelClientReady } = useZeroDev()
-
-    ////// Auth props
-    const { user } = useAuth()
+    const { address: kernelClientAddress, isKernelClientReady, handleLogin } = useZeroDev()
 
     ////// BYOW props
     const { address: wagmiAddress, isConnected: isWagmiConnected } = useAccount()
 
     ////// Selected Wallet
-    const [selectedWallet, setSelectedWallet] = useState<interfaces.IWallet | undefined>(undefined)  // TODO: this is the var that should be exposed for the app to consume, instead of const { address } = useAccount() anywhere
+    const [selectedWallet, setSelectedWallet] = useState<interfaces.IWallet | undefined>(undefined)  // TODO: this is the var that should be exposed for the app to consume, instead of const { address } = useWallet() anywhere
 
     ////// Wallets
     const { data: wallets } = useQuery({
```

Comment:
Sounds good, let's wait until we're actually getting the user (we're not right now) so that the user.accounts are filled. Otherwise we'll break the ui & render current commit untestable